contracts/SimpleNftMarketplace.sol

```diff
@@ -92,11 +97,23 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
     return _listings[listingId].buyer == address(0) && _listings[listingId].seller != address(0);
   }
 
-  function isBlacklistedUser(address userAddress) external view returns (bool isBlacklisted) {}
+  function isBlacklistedUser(address userAddress) external view returns (bool isBlacklisted) {
+    if (_isBlacklistedUser(userAddress) == true) {
+      return true;
+    }
```

Comment:
_isBlacklistedUser(userAddress)  already return bool
so remove the if and just add
```solidity
return _isBlacklistedUser(userAddress);
```